webapp/client/cypress/integration/aussergBela.spec.js

```diff
@@ -0,0 +1,118 @@
+describe("PersonAHasDisability", () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  context("with no data", () => {
+    beforeEach(() => {
+      cy.visit("/lotse/step/ausserg_bela");
+    });
+
+    it("Should redirect back to select_stmind page", () => {
+      cy.location().should((loc) => {
+        expect(loc.pathname.toString()).to.contain("/lotse/step/select_stmind");
+      });
+      cy.get("div[role=alert]").should("exist");
+    });
+  });
+
+  context("with aussergBela selected but no disability data", () => {
+    beforeEach(() => {
+      cy.request("POST", "/testing/set_data/form_data", {
+        stmind_select_ausserg_bela: true,
+      });
+
+      cy.visit("/lotse/step/ausserg_bela");
+    });
+
+    it("Should show standard Aufwendungs inputs", () => {
+      cy.get("#stmind_krankheitskosten_summe").should("exist");
+      cy.get("#stmind_krankheitskosten_anspruch").should("exist");
+
+      cy.get("#stmind_pflegekosten_summe").should("exist");
+      cy.get("#stmind_pflegekosten_anspruch").should("exist");
+
+      cy.get("#stmind_bestattung_summe").should("exist");
+      cy.get("#stmind_bestattung_anspruch").should("exist");
+
+      cy.get("#stmind_aussergbela_sonst_summe").should("exist");
+      cy.get("#stmind_aussergbela_sonst_anspruch").should("exist");
+
+      cy.get("#stmind_krankheitskosten_summe").should("exist");
+      cy.get("#stmind_krankheitskosten_anspruch").should("exist");
+    });
```

Comment:
I decided to put all those fields into one test because this saves some run time and makes it easier to read. In my head this is also just one functionality (show standard fields) that is tested. 
Or do you think we should split those.

Reply:
This is fine for me :)